packages/saber-plugin-query-posts/README.md

```diff
@@ -101,6 +124,35 @@ hello
 
 Then there will be a `/categories/sports` page，`categories` is an array, so you can assign the post to multiple categories.
 
+### Data Injected in Post Pages
+
+You can access the detailed info of the categories of a post page via `this.page.categories`, for instance:
+
+```yaml
+categories:
+  - sports/football
+  - hobby
+```
+
+gives you:
+
+```js
+;[
+  {
+    name: 'sports',
+    permalink: '/categories/sports'
+  },
+  {
+    name: 'sports/fooball',
+    permalink: '/categories/sports/fooball'
+  },
```

Comment:
Should we inject higher level category in this case? i.e. ignore `sports` when there's also `sports/football`?

Reply:
the `permalink` should use full permalink, that's what it calls `permalink`, so keeping parent level category is expected.

Reply:
I mean should we add both `{name:'sports'}` and `{name:'sports/football}` categories here?

Reply:
Hmm, I think it would be useful to have parent category in some use case like porting [this hexo theme](https://probberechts.github.io/hexo-theme-cactus/cactus-classic/public/2013/12/24/categories/), see the category breadcrumb. But how we know that which category should be a child category while another is a parent category?

Reply:
👍